sample/android/build.gradle

```diff
@@ -120,7 +120,7 @@ dependencies {
     androidTestImplementation testLibraries.espresso
     androidTestImplementation testLibraries.espressoContrib
     androidTestImplementation testLibraries.testRules
-    androidTestImplementation testLibraries.screengrab
+    // androidTestImplementation testLibraries.screengrab
```

Comment:
Does not exist anymore because of the bintray shutdown, just commented it out for now so that the library can be built and the sample build was failing before already.

Reply:
Yea, this is one of the examples of not used dependencies that were copy and pasted over when forking from RxRedux.

The sample needs to be cleaned up.